Modules/_elementtree.c

```diff
@@ -1213,12 +1213,8 @@ _elementtree_Element_extend_impl(ElementObject *self, PyTypeObject *cls,
     PyObject* seq;
     Py_ssize_t i;
 
-    seq = PySequence_Fast(elements, "");
+    seq = PySequence_Fast(elements, "argument must be an iterable");
```

Comment:
```suggestion
    seq = PySequence_Fast(elements, "'elements' must be an iterable");
```

When you extend a list, it says `TypeError: "int" object is not iterable`. Since we are removing the type information, I'd suggest at least having the parameter name in the message.